### ISSUE TITLE
[DESIGN] Update minimal header component in Figma

### ISSUE THREAD

**Issue by cgednalske** [department-of-veterans-affairs/va.gov-team#84369]
## Background

FE is switching to the official minimal header component, so we need to make sure Figma matches.

## Tasks
- [ ] Update Check-in and Pre-check-in Figma files with "minimal" version of header (Design files and any features that have not already been released

## AC
- [ ] Pre-check-in and Check-in files are updated with new component

**Comment by benbrasso-agile6** [comment]
@loripusey at standup, we talked about moving this out of this sprint. For example, it's going through design and content changes via collab cycle - https://dsva.slack.com/archives/CMNQT72LX/p1718648595972899

Fine with moving this ticket out of sprint? Zach is going to work on finishing up our Figma reorg.

**Comment by loripusey** [comment]
Moved it back to the backlog

**Comment by benbrasso-agile6** [comment]
@loripusey Shoot, I meant to put my comment on the medication review ticket - https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/82909

**Comment by loripusey** [comment]
So, the 82909 ticket is the one you want taken out of the sprint, not this ticket?

**Comment by benbrasso-agile6** [comment]
Yes. 84369 was not actually in this sprint. I think it was sitting in Next Sprint. Zach is going to work on "internal" work to get our Figma design files straightened up so we can be more efficient. Because it's "internal," it does not have a ticket, but me, Christina and Zach have been meeting to discuss the tasks that need to be complete.

**Comment by loripusey** [comment]
I swapped them

**Comment by benbrasso-agile6** [comment]
All updated
